Eatery/Views/Eateries/CampusEateryStationTableHeaderView.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  CampusEateryStationTableHeaderView.swift
+//  Eatery
+//
+//  Created by Ethan Fine on 2/27/20.
+//  Copyright © 2020 CUAppDev. All rights reserved.
+//
+
+import UIKit
+
+class CampusEateryStationTableHeaderView: UITableViewHeaderFooterView {
+    
+    private var headerView: UIView!
```

Comment:
If we're initializing these views in `init` then there's not much of a point in having them be implicitly unwrapped.